### PR TITLE
Allow opt-in to this workflow by team.

### PR DESCRIPTION
**Pull request by lerebear**
This adds support for a `TEAMS` configuration value (provided via an environment variable), which can be used to opt all the members of the [team(s)](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) into this workflow.

The `TEAMS` value should be provided as a space-delimited string of team slugs. Each team is assumed to belong to the organization that owns the repository that acts as the base of the newly opened pull request. Using the configuration value without the assumed owning organization will cause the workflow to fail.

## TODO

- [x] Validate that this works when integrated with a real workflow
- [x] ~~Release this in a new minor version of this action~~ (see https://github.com/glortho/pr-size-helper-action/pull/1#issuecomment-1515566320)

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -118,6 +118,29 @@ const ensureLabelExists = async (octokit, repo, owner, name, color) => {
...
+        per_page: 100,
+      }
+    )
+
+    for await (const { data: members } of memberIterator)   {
+      for (const member of members) {
+        result.push(member.login)
+      }
+    }
+  }
```
Comment (glortho) [inline src/utils.js:141]: I could be wrong 🤔, but I believe this `await` will hold up iteration of `teams` as well as pages. Is that the desired behavior? That could be appropriate for rate limiting, but unless we strongly suspect it's an issue we can decrease runtime (significantly for multiple paginated teams) by fetching and processing teams concurrently, along the lines of this quasi-pseudo-code 😄:

```javascript
const fetchTeams = async (octokit, organization, teams) => {
  const results = await Promise.allSettled(
    teams.map(
      slug => fetchTeamMembers(octokit, organization, slug)
    )
  )
  return [...results]
}
```

**Review by glortho — COMMENTED** [review]
Nice ✨! I had one optimization question/suggestion but it otherwise looks good to me.

File: src/utils.js (hunk 1 above)
Comment (lerebear) [reply src/utils.js:141]: Good idea! Let me test this to make sure that it works; if it does I will commit it. I copied the previous code from an Octokit example, but I agree that we should avoid the unnecessary serialized behaviour if possible.

Comment (lerebear) [reply src/utils.js:141]: Fixed in 9335288d2abd6b1560d8240fff31eecb11af9a47.

**Comment by lerebear** [comment]
> * [x]  Validate that this works when integrated with a real workflow

This has been completed 🎉 

> * [ ] Release this in a new minor version of this action

@glortho, would you mind merging this and creating the new release? I don't have write permissions on this repo.

**Comment by glortho** [comment]
Nice implementation of that change! ✨ 🚀

**Pull request merged**